lib/screens/oneOnOne/update_1on1_view.dart

```diff
@@ -751,7 +753,7 @@ class _UpdateOneoneOneViewState extends State<UpdateOneoneOneView> {
       maxRadius: 58.0,
       foregroundImage: CachedNetworkImageProvider(selectedEmployee.avatarAttachmentUrl ?? ""),
       child: Text(
-        getInitials(selectedEmployee.name ?? "No Particiapnt", 2),
+        getInitials(selectedEmployee.name ?? "No Particiapnt", 2).toUpperCase(),
```

Comment:
@sriLatha2328 toUpperCase function to be moved inside the getInitials function so that it handles the upper cases wherever used.